### PR TITLE
i18n(tr): add css-and-tailwind

### PR DESCRIPTION
**Pull request by erkanaltinors**
**What kind of changes does this PR include?**

Updates of Starlight docs site content

**Description**

This pull request;

- Adds translated in Turkish `css-and-tailwind.mdx` file into project.
- Fixes typo in `docs/tr/environmental-impact.mdx` file into project.

### CODE REVIEW

File: docs/src/content/docs/tr/guides/css-and-tailwind.mdx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+			inlineCode: 'Satır için kod belirgin arkaplana sahiptir.',
+		},
+	}}
+>
+	<Fragment slot="css-docs">
+	Bu temayı sitenize uygulamak için, [isteğe uyarlanmış CSS
+		dosyası içine](#özel-css-stilleri) aşağıdaki CSS'i dahil edin ve projenize ekleyin.
+	</Fragment>
+	<Fragment slot="tailwind-docs">
+	Aşağıdaki [Tailwind yapılandırma dosyasına ait](#tailwind-ile-starlightı-stillendirme) örnek, yapılandırma nesnesi içerisinde oluşturulmuş `accent` ve `gray` renk paletlerini dahil eder.
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/css-and-tailwind.mdx:292]: I think this sentence is missing the `theme.extend.colors` reference that exists in the English version.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation!

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and except a missing part of a sentence in the last part, I think everything looks perfect 👍

File: docs/src/content/docs/tr/guides/css-and-tailwind.mdx (hunk 1 above)
Comment (erkanaltinors) [reply docs/src/content/docs/tr/guides/css-and-tailwind.mdx]: Hello @HiDeoo , you are right! I've translated the sentence you mentioned.

**Review by HiDeoo — APPROVED** [review]
Amazing, thanks for the fast update, let's merge this one 🚀

**Pull request merged**